src/docbuilder/rustwide_builder.rs

```diff
@@ -867,6 +867,11 @@ mod tests {
                 .map(|v| v.as_str().unwrap().to_owned())
                 .collect();
             targets.sort();
+            assert_ne!(
+                std::env::var("DOCSRS_INCLUDE_DEFAULT_TARGETS").unwrap(),
```

Comment:
Can we set this in the test suite rather than requiring the user to set it?

Reply:
I wound up fixing this in a drive-by fix over here: https://github.com/rust-lang/docs.rs/pull/1892/files#diff-2d6b4b8b3ed45b3dfd5b9f67389371c1f06362ddc510e3add1934c0d7d614f94R282